backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -241,40 +236,36 @@ async function handleShoot (shoot) {
   Until this is the case we need to workaround this by creating an ingress (e.g. with the respective certmanager annotations) so that a proper certificate is presented for the kube-apiserver.
   https://github.com/gardener/gardener/issues/1413
 */
-async function ensureTrustedCertForShootApiServer ({ gardenClient, coreClient, namespace, name }) {
-  const shootResource = await shoots.read({ gardenClient, namespace, name })
-
+async function ensureTrustedCertForShootApiServer (client, shootResource) {
```

Comment:
revert change and make sure to read the latest resource version

Reply:
✔️